images/ose-etcd-operator.yml

```diff
@@ -0,0 +1,20 @@
+content:
+  source:
+    dockerfile: Dockerfile.rhel
+    git:
+      branch:
+        target: openshift-{MAJOR}.{MINOR}
+      url: git@github.com:openshift/etcd-operator.git
+from:
+  builder:
+  - stream: golang
+  member: openshift-enterprise-base
+labels:
+  License: Apache 2.0
+  io.k8s.description: etcd operator creates/configures/manages etcd clusters atop Kubernetes.
+  io.k8s.display-name: etcd operator
+  io.openshift.tags: etcd operator
+name: openshift/ose-etcd-operator
+owners:
+- sbatsche@redhat.com
```

Comment:
I think we're hoping to have team mailing lists here, not individual contributors.